astropy/modeling/tests/test_separable.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Test separability of models.
+
+"""
+
+from astropy.modeling import models
+from astropy.modeling.models import Mapping
+import pytest
+import numpy as np
+from numpy.testing import utils
```

Comment:
`from numpy.testing import assert_allclose` works for me. Is there a reason `utils` needs to be imported at all? I am afraid this might be confusing down the road because there is also `utils` in `astropy` itself.